speed/speed.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main(){
+	var n,km,m float64
+	fmt.Println("This program performs km/h <-> m/s speed conversion.
```

Comment:
What if a user inputs something different than `1` and `2`, e.g. `3`? Your program should handle this case as well.